pyproject.toml

```diff
@@ -21,6 +21,7 @@ dependencies = [
     "deprecated",
     "ipykernel", # Support for Jupyter notebooks
     "scipy", # kdtree
+    "lsst-sphgeom", # To handle spherical sky polygons
```

Comment:
Is this package conda-installable? No worries this moment if not, but it's good to know.

Reply:
Unfortunately it isn't, but considering @mjuric has taken the initiative of creating the recipe to make lsdb conda installable (https://github.com/conda-forge/staged-recipes/pull/24833) I might try to do the same for sphgeom.